Dockerfile

```diff
@@ -127,6 +127,10 @@ RUN /root/.bitrise/tools/stepman update
 
 COPY ./ssh/config /root/.ssh/config
 
+# ------------------------------------------------------
+# --- Install Rubygems
+
+RUN gem update --system --no-document
```

Comment:
The only thing is, it's not "install rubygems" but "update rubygems" (spec/database) ;)